### PR TITLE
Professional recognition fixes

### PR DESCRIPTION
**Pull request by David-Feetenby**
Fix three months warning appearing before contact info

### CODE REVIEW

File: app/views/shared/eligible_region_content_components/_professional_recognition.html.erb [hunk 1]
```diff
@@ -37,6 +37,29 @@
...
+
+<% if region.status_check_online? || region.sanction_check_online? %>
+  <p class="govuk-body">
+    As your education department or authority has an online register of teachers,
+    you’ll also need to provide any reference numbers we need to check your record.
+  </p>
+<% end %>
+
+<% unless FeatureFlags::FeatureFlag.active?(:application_work_history) && !region.application_form_skip_work_history %> 
+  <p class="govuk-body"><%= proof_of_recognition_description_for(region:) %></p>
```
Comment (thomasleese) [inline app/views/shared/eligible_region_content_components/_professional_recognition.html.erb:52]: I don't think we'd want this section to appear above the online status/sanction information aa it looks like the online sentence would now be broken up by the "work history" section.

**Review by steventux — APPROVED** [review]
Just seen @thomasleese 's comment - maybe add a screenshot to the PR?

File: app/views/shared/eligible_region_content_components/_professional_recognition.html.erb (hunk 1 above)
Comment (David-Feetenby) [reply app/views/shared/eligible_region_content_components/_professional_recognition.html.erb:52]: I'll move it above the sentence 👍

File: app/views/shared/eligible_region_content_components/_professional_recognition.html.erb [hunk 2]
```diff
@@ -37,6 +37,29 @@
...
+    <li>school city</li>
+    <li>school country</li>
+    <li>your job role</li>
+    <li>start date</li>
+    <li>end date</li>
+  </ul>
+<% end %>
+
+<% if region.sanction_check_written? %>
+  <p class="govuk-body"><%= t("eligibility_interface.three_months_before_applying") %></p>
```
Comment (thomasleese) [inline app/views/shared/eligible_region_content_components/_professional_recognition.html.erb:53]: Does this sentence belong above the work history as well? It looks like it's more connected to the written statement than the work history.

**Pull request merged**